e2e/testsuite/grpc_query.go

```diff
@@ -132,6 +153,17 @@ func (s *E2ETestSuite) QueryClientStatus(ctx context.Context, chain ibc.Chain, c
 	return res.Status, nil
 }
 
+// QueryCurrentPlan queries the currently scheduled plans, if any
+func (s *E2ETestSuite) QueryCurrentPlan(ctx context.Context, chain ibc.Chain) (upgradetypes.Plan, error) {
```

Comment:
[nit] I think `QueryCurrentUpgradePlan` is a little more explicit, we don't have the context that it is the `upgrade query client` that is being used at the `E2ETestSuite` level.